services/filemanager/s3manager.go

```diff
@@ -74,6 +75,32 @@ func (manager *S3Manager) Upload(file *os.File, prefixes ...string) (UploadOutpu
 	return UploadOutput{Location: output.Location, ObjectName: fileName}, err
 }
 
+func (manager *S3Manager) GetStorageDateFormat(prefixes ...string) (dateFormat string, err error) {
```

Comment:
can we make this func common to all filemanagers? In the func we call `ListFilesWithPrefix` for corresponding filemanager implementation. If response is empty string, we can fallback to new default, otherwise parse and it determine format. 
Can modify `ListFilesWithPrefix` in all implementations to return location without s3 or gcs prefix etc to make it generic